### PR TITLE
Bug 1169340 - Supplemental linting to use correct js indentation

### PR DESCRIPTION
**Pull request by MikeLing**
I fix some code style problem for bug-1169340 which has been merged. Follow French's advice, I use JSLint to find my style issue. But I found it's a little different from original code style of this file(like }); looks like code style mistake in JSLint). So, please tell me if you found some places I should improve, thank you:)
@tojonmz

[Review on Reviewable](https://reviewable.io/reviews/mozilla/treeherder/614)

### CODE REVIEW

**Comment by tojonmz** [comment]
@MikeLing I defer to @jmaher but unless he or @wlach doesn't wish to back out and revert PR https://github.com/mozilla/treeherder/pull/588 (in which case this one would represent the *entire* change rather than corrections) a reminder it's probably good to use the same leading "Bug nnn - " message style here :)

eg. if it just indentation fixes
`Bug 1169340 - Supplemental linting to use correct js indentation`

if 588 is reverted and this PR represents the entire change
`Bug 1169340 - Validate perf compare revisions before loading`

And which ever work this represents, this new PR should also be added as an attachment to the bug.

File: ui/partials/perf/comparechooserctrl.html [hunk 1]
```diff
@@ -21,8 +21,8 @@
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group has-error">
-            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
-            <br/>
+             <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+             <br/>
```
Comment (tojonmz) [inline ui/partials/perf/comparechooserctrl.html]: Please see comment https://github.com/mozilla/treeherder/pull/588/files#r31883014 in the old PR regarding this tag.

**Comment by tojonmz** [comment]
@MikeLing please see my comments in the outdated diffs above, if you are pushing right now.

File: ui/js/compareperf.js [hunk 2]
```diff
@@ -22,18 +22,20 @@ perf.controller('CompareChooserCtrl', [
...
-              function(resultSets) {
-               $scope.newRevisionError = undefined;
-               if($scope.originalRevisionError == undefined && $scope.newRevisionError == undefined) {
-                $state.go('compare', {
-                originalProject: $scope.originalProject.name,
-                originalRevision: $scope.originalRevision,
-                newProject: $scope.newProject.name,
-                newRevision: $scope.newRevision });        
+            function (resultSets) {
+              $scope.newRevisionError = undefined;
```
Comment (tojonmz) [inline ui/js/compareperf.js]: I believe this should be also indented 4 spaces, not 2. So it is aligned with the if statement below it.

File: ui/partials/perf/comparechooserctrl.html [hunk 3]
```diff
@@ -21,8 +21,8 @@
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group has-error">
-            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
-            <br/>
+              <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+              <br/>
```
Comment (tojonmz) [inline ui/partials/perf/comparechooserctrl.html]: Please see my comments in https://github.com/mozilla/treeherder/pull/588#discussion_r31883014 regarding this tag.

**Comment by wlach** [comment]
I think these changes are worth doing, but we should file a new bug to track them and request review of me on it seperately (since this really does not have anything to do with the comparison chooser, which is what bug 1169340 was about). @MikeLing: could you go ahead and do that? You can file a new bug in perfherder at this URL:

https://bugzilla.mozilla.org/enter_bug.cgi?product=Tree%20Management&component=Perfherder

I'd just title the bug something like "Fix minor indentation problems in perfherder"

Also modify the commit message of the actual code to reference the new bug.

File: ui/partials/perf/comparechooserctrl.html [hunk 4]
```diff
@@ -12,7 +12,7 @@
           <label for="original-revision-input">Revision</label>
           <div class="form-group has-error">
             <input id="original-revision-input" maxlength="12" class="form-control" ng-model="originalRevision">
-            <br/>
+            <br>
```
Comment (wlach) [inline ui/partials/perf/comparechooserctrl.html]: <br/> indicates a singular element (vs. <br> which implies there is a closing </br> somewhere, which there isn't), we want to keep that syntax. Please take out this change -- it should actually be the input element which has a closing "/" on it, in fact. i.e.:

    <input id="original-revision-input" maxlength="12" class="form-control" ng-model="originalRevision"/>

File: ui/partials/perf/comparechooserctrl.html [hunk 5]
```diff
@@ -24,8 +24,8 @@
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
             <div class="form-group has-error">
-            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
-            <br/>
+              <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+              <br>
```
Comment (wlach) [inline ui/partials/perf/comparechooserctrl.html]: See comments above on <br> vs <br/> and <input> vs <input/>

**Comment by wlach** [comment]
Merged! In the future for cases like this where we change the bug # please update the title of the PR as well to reduce confusion. :)

**Pull request merged**